### ISSUE TITLE
Add accessible/tooltip labels for all buttons

### ISSUE THREAD

**Issue by vkbo** [vkbo/novelWriter#2338]
Many buttons on the GUI do not have a label. Mainly because they are only icon buttons. A screen reader cannot explain what the button is for, so these need a text. The same text can be used for a tooltip, which will help in general.
